client/gutenberg/extensions/simple-payments/editor.js

```diff
@@ -20,7 +20,9 @@ registerBlockType( 'a8c/simple-payments', {
 	category: 'jetpack',
```

Comment:
Should the description above include a link or be tweaked otherwise?

Reply:
@michaelarestad @mapk We don’t have an established pattern to include support doc -links in blocks, right? 

Seems like it could be a simple component always tossed into the sidebar.